client/src/components/layers/EditAnnotationLayer.vue

```diff
@@ -96,33 +96,33 @@ export default {
         }
         annotation.editHandleStyle({ handles: { rotate: false } });
         if (this.editing) {
-          this.$geojsLayer.mode("edit", annotation);
+          this.$geojsLayer.mode('edit', annotation);
           this.$geojsLayer.draw();
         }
       } else if (this.editing) {
         this.changed = true;
-        if (typeof this.editing !== "string") {
+        if (typeof this.editing !== 'string') {
           throw new Error(
             `editing props needs to be a string of value ${geo
               .listAnnotations()
-              .join(", ")} when geojson prop is not set`
+              .join(', ')} when geojson prop is not set`,
```

Comment:
Is this more readable?
![image](https://user-images.githubusercontent.com/61746913/79347966-558a2380-7f02-11ea-8187-3872370f2866.png)